driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/getuid.bpf.c

```diff
@@ -14,19 +15,19 @@ int BPF_PROG(getuid_e,
 	     struct pt_regs *regs,
 	     long id)
 {
-	struct ringbuf_struct ringbuf;
```

Comment:
Based on discussion in [comment](https://github.com/falcosecurity/libs/pull/752#discussion_r1042446028) in #752, let's keep the ringbuf here. So you can skip this commit.